forum/api_utils.py

```diff
@@ -0,0 +1,47 @@
+"""Forum Utils."""
```

Comment:
Can we rename that module? Otherwise all non-API functions will end up in this "utils" module, which is not a very descriptive name. Maybe api/threads.py?

Reply:
you means, a hierarchy like
```
api.py
api/threads.py
api/comments.py
...
```

i think we can then just move the API functions to their respective views then?
like and import these APIs in edx-platform from below path.
```
forum/views/pins.py
forum/views/threads.py
forum/views/comments.py
```

Reply:
Almost:

```
api/__init__py
api/threads.py
api/comments.py
...
```